src/thor/bidirectional_astar.cc

```diff
@@ -549,11 +557,17 @@ inline bool BidirectionalAStar::ExpandReverseInner(GraphReader& graphreader,
   float sortcost =
       newcost.cost + astarheuristic_reverse_.Get(t2->get_node_ll(meta.edge->endnode()), dist);
 
+  // There is an edge case where we may encounter only_restrictions with edges being
```

Comment:
consider moving this lengthy comment into the PathAlgorithm method that sets not_thru pruning.